app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginViewModel.java

```diff
@@ -76,8 +93,8 @@ public void setBaseUrl(String url, boolean shouldSetDefaultUrl) {
         interceptor.setInterceptor(baseUrl);
     }
 
-    public Login getLogin() {
-        return login;
+    public LiveData<Login> getLogin() {
+        return decryptedLogin;
```

Comment:
You removed all the logic, where are you loading the decrypted data?

Reply:
I pushed by mistake